### ISSUE TITLE
VisualDB client errors

### ISSUE THREAD

**Issue by geertdepuydt** [dolthub/dolt#6711]
Trying out VisualDB (https://visualdb.com/) as a front-end to my local Dolt database (dolt latest 1.16.3).
VisualDB connects successfully as a MySQL client, but I get a lot of errors trying to read (update, write) table data, like:
```
2023-09-25T13:53:10+02:00 INFO [conn 10] NewConnection {DisableClientMultiStatements=false}
2023-09-25T13:53:10+02:00 INFO [no conn] Cannot read client handshake response from client 10 (35.203.211.66:59084): Code: INTERNAL
invalid sequence, expected 1 got 0
, it may not be a valid MySQL client {}
```
Just posting it here, on the off-chance this is a relatively easy fix.
I have also contacted VisualDB just in case the fix is on their end.

**Comment by timsehn** [comment]
Thanks. We will look at this today or maybe tomorrow.

**Comment by fulghum** [comment]
Hi @geertdepuydt, thank you for letting us know about this compatibility issue so we can dig into it! 

I've got a local Dolt sql-server running and have been able to connect to it from VisualDB. I haven't been able to trigger the `Cannot read client handshake response` error yet though. To try and repro the issue, I did this so far:
- imported a csv to create a table
- created a simple query to select everything from that table
- created a spreadsheet from that form and have been manually editing in the spreadsheet UI
- I also tried a couple of queries from the SQL Scratchpad tab in the Connections view; from the Dolt sql-server logs, it looks like they worked, but Visual DB doesn't seem to show results here 

I'll keep poking on this and seeing if I can repro the same error. If you think of any more details on how you triggered this, let us know and we'll keep digging deeper!
